NAMESPACE

```diff
@@ -108,6 +109,7 @@ export(asLabel)
 export(createWatermarkGrob)
 export(exportPlot)
 export(exportPlotConfigurationCode)
+export(geom_tlf_point)

```

Comment:
let's call it `geomTlfPoint `(or `geomTLFPoint`) to be consistent with the overall naming